src/components/comment/index.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { toast } from 'react-toastify';
+import TextAreaInput from '../TextAreaInput';
```

Comment:
@oyedejipeace If this import is from a directory, please refer to above comment.